k8s/namespaces/wa/wa-messages-find-problem-messages/aat-00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-messages-find-problem-messages
+spec:
+  values:
+    job:
+      schedule: "0,30* * * *"
```

Comment:
Think you need a space between, otherwise it's invalid

Reply:
got it